src/test/java/org/opensearch/neuralsearch/query/HybridQueryIT.java

```diff
@@ -578,6 +579,54 @@ public void testRequestCache_whenMultipleShardsQueryReturnResults_thenSuccessful
         }
     }
 
+    @SneakyThrows
+    public void testIndexAlias_whenHybridQueryAndIndexAliasHasFilter_thenSuccess() {
```

Comment:
@martin-gaievski can we add a test when there is both nested fields and index alias filters.

Reply:
ack

Reply:
Extend this test case, now both criteria are there - nested fields in index mapping and alias with filter